src/utils.ts

```diff
@@ -109,6 +121,65 @@ export class Utils {
         }
     }
 
+    /**
+     * Retrieves the application key from input or .jfrog configuration file.
+     *
+     * This method attempts to retrieve the application key from the GitHub Action input.
+     * If the input is not provided, it reads .jfrog configuration file from the file system.
+     * If the configuration file exists and contains the application key, it returns the key.
+     * If the configuration file does not exist or does not contain the application key, it returns an empty string.
```

Comment:
This description is wrong, because the function should only read the application key from the confab file.